GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -137,6 +135,40 @@ - (nullable NSString *)openIDRealm {
   return [_authState.lastTokenResponse.request.additionalParameters[kOpenIDRealmParameter] copy];
 }
 
+- (NSString *)idTokenString {
```

Comment:
In general, I think it's useful to avoid using type information in the name of a method, property, or variable. I typically do so only to avoid a collision with something other name. Another use case that I think makes sense is if your method provides a `String` representation of another type.

Reply:
Yes, this is to avoid the naming conflict with idToken which is a GIDToken type.

Reply:
Is there actually a naming conflict? It looks like you removed the getter for `idToken` and replaced it with `idTokenString`.

Reply:
I see the property for `idToken` now. Sorry for missing it. :) 

Given that this getter works with `_authState.lastTokenResponse.idToken`, I'd suggest not naming it `idTokenString` (i.e., you're not going through the property `idToken` on `GIDGoogleUser`). Maybe rename this to `lastTokenResponseIdToken` or something like that.

This suggestion applies here and throughout.